test/invariant/transitions_test.go

```diff
@@ -294,19 +294,9 @@ func pickActorAndTopicIdForStateTransition(
 		}
 		topics := findActiveTopicsAtThisBlock(m, data, blockHeightNow)
 		if len(topics) > 0 {
-			for i := 0; i < 10; i++ {
-				randIndex := m.Client.Rand.Intn(len(topics))
-				topicId := topics[randIndex].Id
-				worker, err := data.pickRandomWorkerRegisteredInTopic(m.Client.Rand, topicId)
-				if err != nil {
-					continue
-				}
-				reputer, err := data.pickRandomReputerRegisteredInTopic(m.Client.Rand, topicId)
-				if err != nil {
-					continue
-				}
-				return true, worker, reputer, nil, topicId
-			}
+			randIndex := m.Client.Rand.Intn(len(topics))
+			topicId := topics[randIndex].Id
+			return true, UnusedActor, UnusedActor, nil, topicId
```

Comment:
just curious, why `UnusedActor` here?

Reply:
doInferenceAndReputation doesn't pick a particular actor to do the inference and reputation

it uses every registered actor in the topic, makes every single person submit a worker and reputer payload

yes this could be changed in a future release for better coverage, but at a minimum is how the fuzzer has worked historically